timeline.js

```diff
@@ -1,51 +1,53 @@
-import { db } from "./dbInstance.js";
+import { db } from './dbInstance.js'
+
+let hasLoaded = false
 
 class ReaderTimeline extends HTMLElement {
-  constructor() {
-    super();
-    this.actorUrls = [
-      "https://staticpub.mauve.moe/about.jsonld",
-      "https://hypha.coop/about.jsonld",
-      "https://prueba-cola-de-moderacion-2.sutty.nl/about.jsonld",
-    ];
-    this.processedNotes = new Set(); // To keep track of notes already processed
+  connectedCallback () {
+    this.initializeDefaultFollowedActors().then(() => this.initTimeline())
   }
 
-  connectedCallback() {
-    this.initTimeline();
+  async initializeDefaultFollowedActors () {
+    const defaultActors = [
+      'https://social.dp.chanterelle.xyz/v1/@announcements@social.dp.chanterelle.xyz/',
+      'https://hypha.coop/about.jsonld',
+      'https://sutty.nl/about.jsonld'
+      // "https://akhilesh.sutty.nl/about.jsonld",
+      // "https://staticpub.mauve.moe/about.jsonld",
+    ]
+
+    // Check if followed actors have already been initialized
+    const hasFollowedActors = await db.hasFollowedActors()
+    if (!hasFollowedActors) {
+      await Promise.all(
+        defaultActors.map(async (actorUrl) => {
+          await db.followActor(actorUrl)
+        })
+      )
+    }
   }
 
-  async initTimeline() {
-    this.innerHTML = ""; // Clear existing content
+  async initTimeline () {
```

Comment:
We should have a `loadMore` button and keep track of the "skip/limit" for paging.